src/tests/user.rs

```diff
@@ -510,12 +514,13 @@ fn test_confirm_user_email() {
 
     // Simulate logging in via GitHub. Don't use app.db_new_user because it inserts a verified
     // email directly into the database and we want to test the verification flow here.
+    let email = "cow@mammals@milk";
```

Comment:
I'm curious, why is the value different here? Any particular reason, and any reason why it's not a valid email address (two `@`s)?